### DISCUSSION TITLE
Remove id’s from templates

### DISCUSSION THREAD

**Discussion by zgeist** [GrapesJS/grapesjs#4399] [Ideas]
For example: https://github.com/artf/grapesjs/blob/dev/src/style_manager/view/PropertyStackView.js#L20
or https://github.com/artf/grapesjs/blob/dev/src/selector_manager/view/ClassTagView.ts#L13 or same

Create items with the same ID, of course in the main it works fine, but sometimes you may have problems with it

How realistic is it to drop of IDs in templates?

**Comment by artf** [comment]
Yeah, I'd say those are actually quite old templates, switching to classes would be way better. I'll be happy to merge a PR :)
